pages/pipelines/defining_steps.md.erb

```diff
@@ -147,9 +147,17 @@ Differentiating between `broken`, `skipped` and `canceled` states:
 
 <%= render_markdown 'pipelines/job_states' %>
 
+Each job in a build also has a footer that displays exit status information. It may include exit signal reason, which indicates which indicates whether the Buildkite agent stopped or the job was cancelled..
```

Comment:
```suggestion
Each job in a build also has a footer that displays exit status information. It may include exit signal reason, which indicates which indicates whether the Buildkite agent stopped or the job was cancelled.
```